src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -103,24 +105,37 @@ private void updateOutOfSyncInvoiceLine(InvoiceLine invoiceLine) {
   /**
    * Calculate invoice line total and compare with original value if it has changed
    *
-   * @param invoiceLineFromStorage invoice line to update totals for
-   * @return {code true} if grand total value is different to original one
+   * @param invoiceLine invoice line to update totals for
+   * @param invoice invoice record
+   * @return {code true} if any total value is different to original one
    */
-  public CompletableFuture<Boolean> reCalculateInvoiceLineTotals(InvoiceLine invoiceLineFromStorage) {
+  private boolean reCalculateInvoiceLineTotals(InvoiceLine invoiceLine, Invoice invoice) {
```

Comment:
method name `isInvoiceLineTotalsRecalculationRequired` is more meaningful from my opinion